src/pages/Chat/Chat.styled.jsx

```diff
@@ -135,7 +135,9 @@ export const MessageBox = styled.div`
 export const Avatar = styled.img`
   display: inline-block;
   float: ${({ type }) => (type === 'client' ? `left` : `right`)};
-  width: 14vw;
+  width: 4vw;
+  min-width: 40px;
+  max-width: 50px;
```

Comment:
What about something like 1200px?

Reply:
I think 1200px is so big for the avatar image

Reply:
Ah, I thought it was for the container

OK, if is only the image is ok, can you @Katsu08 make a task to do the same treatment to container?